RELEASE-NOTES.txt

```diff
@@ -1,3 +1,13 @@
+1.25.0
+------
+* New block: Cover
+* Improve icon on the "Take a Video" media option
+* Removed the dimming effect on unselected blocks
```

Comment:
I was a bit confused about this dimming of unselected blocks. Then I figured out that it was referring to inner-blocks. Could we clarify this entry? 
I'd go with adding a `"Inner blocks: ..."` at the beginning of the sentence to make it clearer.

cc @pinarol

Reply:
The dimming effect weren't only applied to inner blocks, it was applied to every type of unselected block. But you'd need to select an inner block to see the dimming. It is hard to describe from a users perspective, and not sure if that's a critical detail to give the users.

Reply:
But the dimming happened when selecting inner blocks, right?
I thought it might also be confusing for our excellence wranglers in the testing phase.

cc @designsimply

Reply:
If we want to update, I suggest being more specific instead of just adding "Inner blocks:" at the beginning.

"Removed the dimming effect on the unselected blocks applied when a block is selected inside a Group or Media-Text block"